Updating messaging to match all Teleport protocols (#10768)
gravitational/teleport: docs/pages/enterprise/fedramp.mdx

---

> title: FedRAMP compliance for Infrastructure access 

Reviewer: ```suggestion
title: FedRAMP Compliance for Infrastructure Access 
```
Per style guide

> title: FedRAMP compliance for SSH, Kubernetes, Databases and Web Apps.

Reviewer: ```suggestion
title: FedRAMP compliance for SSH, Kubernetes, Databases and Web Apps
```

Reviewer: Also, the title might be on the long side now. With the ` | Teleport Docs` suffix, it's 80 chars. Maybe use a shorter title here and this title for `h1` so the title isn't truncated in search results?

Author: Thanks @ptgott . I've added a shorter one in a different commit.

> title: FedRAMP compliance for SSH, Kubernetes, Databases and Web Apps.

Author: ```suggestion
title: FedRAMP compliance for Infrastructure access 
```

> title: FedRAMP compliance for SSH, Kubernetes, Databases, Desktops and Web Apps.

Reviewer 2: I would remove desktops for now as we haven't done any work to verify that desktop access can be FedRAMP compliant.

Author: fixed

> description: How to configure SSH, Kubernetes, Databases,  and Web Apps access to be FedRAMP compliant, including support for FIPS 140-2

Reviewer: ```suggestion
description: How to configure SSH, Kubernetes, database, and web app access to be FedRAMP compliant, including support for FIPS 140-2
```
small grammar tweak
